src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -540,6 +540,22 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     })()
   }, [accountId, accountMetadata, accountType, assetId, bip44Params, chainAdapter, wallet])
 
+  const { data: isTradingActive, refetch: refetchIsTradingActive } = useQuery({
+    ...reactQueries.common.isTradingActive({
+      assetId,
+      swapperName: SwapperName.Thorchain,
+    }),
+    // @lukemorales/query-key-factory only returns queryFn and queryKey - all others will be ignored in the returned object
+    enabled: Boolean(assetId),
+    // Go stale instantly
+    staleTime: 0,
+    // Never store queries in cache since we always want fresh data
+    gcTime: 0,
+    refetchOnWindowFocus: true,
+    refetchOnMount: true,
+    refetchInterval: 60_000,
+  })
```

Comment:
what's the rationale behind repeating these? seems a bit un-dry (moist?)

Reply:
I see the comment above but i dont understand it 😅

Reply:
ok so from the PR description

> This effectively meant we were ditching all the options of isTradingActive, such as gcTime, which is the guts of isTradingActive always getting fresh data. In other words, we thought we were getting fresh data everytime with this hook, but were really using the default 5mn garbage collect time of react-query.

Makes sense. I wonder, could we wrap this somehow to avoid mass copypasta?

Reply:
I guess we could spread common ones with nice and declarative names, or perhaps helper functions - will take a look at doing this!

Reply:
That will make more sense in the PR on top of this one however as this does a full pass through these 🙏🏽